python/ray/dag/compiled_dag_node.py

```diff
@@ -765,7 +765,18 @@ def _preprocess(self) -> None:
         self.input_task_idx, self.output_task_idx = None, None
         self.actor_task_count.clear()
 
-        nccl_actors_p2p: Set["ray.actor.ActorHandle"] = set()
+        # Actors and DAG nodes involved in NCCL P2P send/recv with `transport=nccl`,
+        # in which case the default NCCL group is used. The default NCCL group is the
+        # `custom_nccl_group` in `experimental_compile` if specified. If not, we
+        # first try borrowing another NCCL group in the DAG that is already
+        # initialized. A `_NcclGroup` is created if we can't borrow.
+        default_nccl_actors_p2p: Set["ray.actor.ActorHandle"] = set()
+        default_nccl_nodes_p2p: Set[DAGNode] = set()
+        # Maps each custom NCCL group to the DAG nodes that use it as their P2P
```

Comment:
This comment does not explain anything more than the variable name.

Reply:
Removed.